### PR TITLE
Now self.thrustToWeight is defined in class rocket

### PR DESCRIPTION
**Pull request by Gui-FernandesBR**
This change aims to fix some problems with pull request #36

### CODE REVIEW

File: rocketpy/Rocket.py [hunk 1]
```diff
@@ -267,6 +269,7 @@ def __init__(
         # Calculate dynamic inertial quantities
         self.evaluateReducedMass()
         self.evaluateTotalMass()
+        self.thrustToWeight = self.motor.thrust()/(9.80665*self.totalMass())
```
Comment (giovaniceotto) [inline rocketpy/Rocket.py:272]:
```suggestion
        self.thrustToWeight = self.motor.thrust/(9.80665*self.totalMass)
```

File: rocketpy/Rocket.py [hunk 2]
```diff
@@ -888,6 +891,14 @@ def allInfo(self):
         self.staticMargin()
         self.powerOnDrag()
         self.powerOffDrag()
+        self.thrustToWeight()
```
Comment (giovaniceotto) [inline rocketpy/Rocket.py:894]:
```suggestion
        self.thrustToWeight.plot(lower=0, upper=self.motor.burnOutTime)
```

File: rocketpy/Rocket.py [hunk 3]
```diff
@@ -888,6 +891,14 @@ def allInfo(self):
...
+
+        #ax = plt.subplot(415)
+        #ax.plot(  , self.rocket.motor.thrust()/(self.env.g() * self.rocket.totalMass()))
+        #ax.set_xlim(0, self.rocket.motor.burnOutTime)
+        #ax.set_xlabel("Time (s)")
+        #ax.set_ylabel("Thrust/Weight")
+        #ax.set_title("Thrust-Weight Ratio")
```
Comment (giovaniceotto) [inline rocketpy/Rocket.py:901]: Any reason for these lines to be here for the future?

Comment (Gui-FernandesBR) [reply rocketpy/Rocket.py:901]: these lines would help us in editing figure size for Thrus/Weight plot

File: rocketpy/Rocket.py [hunk 4]
```diff
@@ -267,6 +269,7 @@ def __init__(
         # Calculate dynamic inertial quantities
         self.evaluateReducedMass()
         self.evaluateTotalMass()
+        self.thrustToWeight = self.motor.thrust/(9.80665*self.totalMass)
```
Comment (giovaniceotto) [inline rocketpy/Rocket.py:272]:
```suggestion
        self.thrustToWeight = self.motor.thrust/(9.80665*self.totalMass)
        self.thrustToWeight.setInputs('Time (s)')
        self.thrustToWeight.setOutputs('Thrust/Weight')
```

Comment (Gui-FernandesBR) [reply rocketpy/Rocket.py:272]: Ty so much, now it is working!!!!! 

See an example:

image

**Pull request merged**